docs/release/release-2.0.rst

```diff
@@ -0,0 +1,40 @@
+===========
+Release 2.0
+===========
+
+New Features
+------------
+* Make presto-admin log and configuration directories configurable. They can be
+  set using the environment variables ``PRESTO_ADMIN_LOG_DIR`` and
+  ``PRESTO_ADMIN_CONFIG_DIR``.
+* Change the default configuration directory to ``~/.prestoadmin`` and the
+  default log directory to ``~/.prestoadmin/log``
+* Remove requirement for running and installing presto-admin with sudo.
+  Presto-Admin still uses sudo to execute commands on the Presto nodes.
+* Rename the ``connectors`` directory to ``catalog`` to match the Presto
+  nomenclature
+* Rename the ``connector add`` and ``connector remove``  commands to
+  ``catalog add`` and ``catalog remove``
+* Add experimental support for connecting to a Presto server with internal
+  communication via HTTPS and LDAP, where the HTTP connection is disabled
+* Allow specifying which python to use as an argument to the presto-admin
+  installation script
+* Add ``G1HeapRegionSize=32M`` to the jvm.config defaults as suggested by the
+  Presto documentation
+
+Bug Fixes
+---------
+* Keep the ``node.id`` in Presto's ``node.properties`` file consistent across
+  configuration updates
+* Give Presto configuration files ``644`` permissions
```

Comment:
-> Change Presto configuration files to have 644 permissions.

Give sounds weird.